certs/get_certs.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+working=$1
+domain=$2
+
```

Comment:
I'd add something like:
```
tag=$1
if [ -z "$working" ]; then
  echo "Must pass in the working directory to use for docker image, e.g. $(pwd)"
  exit 1
fi

if [ -z "$domain" ]; then
  echo "Must pass in the domain to use for docker image, e.g. pipelines.dev.data.humancellatlas.org"
  exit 1
fi
```
So if we miss some inputs, it won't throw the error after building the docker. What do you think?